dynamiq/nodes/llms/base.py

```diff
@@ -102,6 +107,7 @@ class BaseLLM(ConnectionNode):
     schema_: dict[str, Any] | type[BaseModel] | None = Field(
         None, description="Schema for structured output or function calling.", alias="schema"
     )
+    response_transformer: ResponseTransformer = ResponseTransformer.DEFAULT
```

Comment:
Let's make it Optional. And apply just when it passes. I think we also should remove the DEFAULT option as by default we return our structure (and DEFAULT it's not clear)